Makefile

```diff
@@ -65,3 +65,7 @@ autoflake:
 
 .PHONY: format
 format: black isort autoflake
+
+.PHONY: unittest
+unittest:
```

Comment:
optional: Could this become `unit-test` to be consistent with https://github.com/linode/go-metadata/pull/31?

Reply:
nice catch